src/libraries/System.Private.Runtime.InteropServices.JavaScript/tests/System.Private.Runtime.InteropServices.JavaScript.Tests.csproj

```diff
@@ -3,11 +3,20 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-Browser</TargetFrameworks>
     <TestRuntime>true</TestRuntime>
-    <WasmXHarnessArgs>$(WasmXHarnessArgs) --engine-arg=--expose-gc --web-server-use-cop</WasmXHarnessArgs>
+    <WasmXHarnessArgs>$(WasmXHarnessArgs) --engine-arg=--expose-gc --web-server-use-cop --no-headless</WasmXHarnessArgs>
+    <!-- Use following lines to write the generated files to disk. -->
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <CompilerGeneratedFilesOutputPath>$(IntermediateOutputPath)\GeneratedFiles</CompilerGeneratedFilesOutputPath>
```

Comment:
As said, I don't think the `IntermediateOutputPath` property is already available here. @ericstj any suggestion on how to better set this property? Maybe setting it in an InitialTarget?

Reply:
@pavelsavara any reason why you can't use the default location: `$(IntermediateOutputPath)/generated`? That way you don't need to specify the path yourself and let roslyn set it: https://github.com/dotnet/roslyn/blob/c01ad7600a0d4cef7196c0fd43bc0132bf0238cf/src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets#L327-L328.

Reply:
Thanks for the tip, will do